libraries/AP_RCProtocol/AP_RCProtocol_MavlinkRadio.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include "AP_RCProtocol_config.h"
+
+#if AP_RCPROTOCOL_MAVLINK_RADIO_ENABLED
+
+#include "AP_RCProtocol_MAVLinkRadio.h"
+
+// constructor
+AP_RCProtocol_MAVLinkRadio::AP_RCProtocol_MAVLinkRadio(AP_RCProtocol &_frontend) :
+    AP_RCProtocol_Backend(_frontend)
+{}
+
+void AP_RCProtocol_MAVLinkRadio::update_radio_rc_channels(const mavlink_radio_rc_channels_t* packet)
+{
+    uint8_t count = packet->count;
+    if (count >= MAX_RCIN_CHANNELS) count = MAX_RCIN_CHANNELS;
```

Comment:
```suggestion
    const uint8_t count = MIN(packet->count, count >= MAX_RCIN_CHANNELS);
```

... our style guide requires braces on every if statement (and line feeds), so this was going to get longer...

Reply:
THX. done (with some correction).